--- RMagick-2.13.4/ext/RMagick/rmmain.c~	2015-03-17 08:49:27.000000000 +0100
+++ RMagick-2.13.4/ext/RMagick/rmmain.c	2015-03-17 08:50:17.893133135 +0100
@@ -1616,7 +1616,7 @@
     /*-----------------------------------------------------------------------*/
 
     SetFatalErrorHandler(rm_fatal_error_handler);
-    SetErrorHandler(rm_error_handler);
+//    SetErrorHandler(rm_error_handler);
     SetWarningHandler(rm_warning_handler);
 
 }
