--- RMagick-4.1.0/lib/rmagick_internal.rb~	2020-03-24 10:30:49.000000000 +0100
+++ RMagick-4.1.0/lib/rmagick_internal.rb	2020-03-24 10:34:54.810077969 +0100
@@ -285,8 +285,7 @@
     # colorization rule
     def alpha(x, y, method)
       Kernel.raise ArgumentError, 'Unknown paint method' unless PAINT_METHOD_NAMES.key?(method.to_i)
-      name = Gem::Version.new(Magick::IMAGEMAGICK_VERSION) > Gem::Version.new('7.0.0') ? 'alpha ' : 'matte '
-      primitive name + sprintf('%g,%g, %s', x, y, PAINT_METHOD_NAMES[method.to_i])
+      primitive 'alpha ' + sprintf('%g,%g, %s', x, y, PAINT_METHOD_NAMES[method.to_i])
     end
 
     # Draw an arc.
