diff -urNp -x '*.orig' rmagick-4.1.0.org/Rakefile rmagick-4.1.0/Rakefile
--- rmagick-4.1.0.org/Rakefile	2022-04-16 10:15:10.279112702 +0200
+++ rmagick-4.1.0/Rakefile	2022-04-16 10:15:10.809107765 +0200
@@ -28,14 +28,6 @@ task :config do
   end
 end
 
-desc 'abort when repo is not clean or has uncommited code'
-task :assert_clean_repo do
-  sh('git diff --exit-code')
-  abort 'Git repo not clean' unless $CHILD_STATUS.success?
-  sh('git diff-index --quiet --cached HEAD')
-  abort 'Git repo not commited' unless $CHILD_STATUS.success?
-end
-
 desc 'build gem'
 task build: [:config] do
   sh 'gem build -V rmagick.gemspec'
@@ -48,24 +40,6 @@ task build: [:config] do
   end
 end
 
-task push_and_tag: [:build] do
-  sh "gem push #{File.join(base, 'pkg', gem_name)}"
-  if $CHILD_STATUS.success?
-    sh "git tag -a -m \"Version #{version}\" #{version_tag}"
-    STDOUT.puts "Tagged #{version_tag}."
-    sh 'git push'
-    sh 'git push --tags'
-  else
-    abort 'tagging aborted pushing gem failed'
-  end
-end
-
-desc 'Release'
-task release: %i[assert_clean_repo push_and_tag]
-
-desc 'Release and build the legacy way'
-task legacy_release: ['legacy:README.html', 'legacy:extconf', 'legacy:doc', 'legacy:manifest', 'release']
-
 namespace :legacy do
   require 'find'
 
diff -urNp -x '*.orig' rmagick-4.1.0.org/rmagick.gemspec rmagick-4.1.0/rmagick.gemspec
--- rmagick-4.1.0.org/rmagick.gemspec	2022-04-16 10:15:10.292445911 +0200
+++ rmagick-4.1.0/rmagick.gemspec	2022-04-16 10:15:10.809107765 +0200
@@ -13,7 +13,7 @@ Gem::Specification.new do |s|
   s.homepage = 'https://github.com/rmagick/rmagick'
   s.license = 'MIT'
 
-  tracked_files = `git ls-files`.split($OUTPUT_RECORD_SEPARATOR)
+  tracked_files = `find ./ -printf "%P\n"`.split($OUTPUT_RECORD_SEPARATOR)
   file_exclusion_regex = %r{\A(doc|benchmarks|examples|spec|lib/rvg/to_c.rb)}
   files         = tracked_files.reject { |file| file[file_exclusion_regex] }
   test_files    = files.grep(%r{^(test|spec|features)/})
