diff -ur RMagick-1.14.1-o/ext/RMagick/rmmain.c RMagick-1.14.1-n/ext/RMagick/rmmain.c
--- RMagick-1.14.1-o/ext/RMagick/rmmain.c	2006-10-21 07:43:23.000000000 -0600
+++ RMagick-1.14.1-n/ext/RMagick/rmmain.c	2006-12-25 16:21:40.000000000 -0700
@@ -1511,8 +1511,6 @@
         ENUMERATOR(AverageInterpolatePixel)
         ENUMERATOR(BicubicInterpolatePixel)
         ENUMERATOR(BilinearInterpolatePixel)
-        ENUMERATOR(FilterInterpolatePixel)
-        ENUMERATOR(IntegerInterpolatePixel)
         ENUMERATOR(MeshInterpolatePixel)
         ENUMERATOR(NearestNeighborInterpolatePixel)
     END_ENUM
diff -ur RMagick-1.14.1-o/ext/RMagick/rmutil.c RMagick-1.14.1-n/ext/RMagick/rmutil.c
--- RMagick-1.14.1-o/ext/RMagick/rmutil.c	2006-10-21 07:41:32.000000000 -0600
+++ RMagick-1.14.1-n/ext/RMagick/rmutil.c	2006-12-25 16:21:56.000000000 -0700
@@ -1587,8 +1587,6 @@
         ENUM_TO_NAME(AverageInterpolatePixel)
         ENUM_TO_NAME(BicubicInterpolatePixel)
         ENUM_TO_NAME(BilinearInterpolatePixel)
-        ENUM_TO_NAME(FilterInterpolatePixel)
-        ENUM_TO_NAME(IntegerInterpolatePixel)
         ENUM_TO_NAME(MeshInterpolatePixel)
         ENUM_TO_NAME(NearestNeighborInterpolatePixel)
     }
